### PR TITLE
Make shark icon smaller

### PR DESCRIPTION
**Pull request by reesericci**
Already merged into master, and truenas 12.0-stable. Filing PR to merge into FreeNAS current release

### CODE REVIEW

**Comment by reesericci** [comment]
#4515 #4487 are the other PR's

**Comment by erinix** [comment]
For 11.3 this is the wrong branch, you would want to use freenas/11.3-stable.

**Comment by reesericci** [comment]
ok

**Comment by reesericci** [comment]
ok done @erinix

File: src/app/pages/account/users/user-form/user-form.component.ts [hunk 1]
```diff
@@ -362,6 +362,11 @@ export class UserFormComponent {
       if (!entityForm.isNew) {
         entityForm.setDisabled('uid', true);
         entityForm.formGroup.controls['username'].setValue(res[0].username);
+        // Be sure namesInUse is loaded, edit it, set username again to force validation
```
Comment (erinix) [inline src/app/pages/account/users/user-form/user-form.component.ts:365]: this isn't the shark icon :)

Comment (reesericci) [reply src/app/pages/account/users/user-form/user-form.component.ts:365]: Oh, did it bring over changes when I change the branch? Oopsies

Comment (reesericci) [reply src/app/pages/account/users/user-form/user-form.component.ts:365]: How do I get rid of them?

Comment (reesericci) [reply src/app/pages/account/users/user-form/user-form.component.ts:365]: How do I get rid of them @erinix ?

**Comment by reesericci** [comment]
@erinix how do I get rid of the changes that occur from switching branches?

**Comment by erinix** [comment]
I would suggest making a new PR and cherry-picking just the commit you want

**Comment by reesericci** [comment]
ok will do

**Pull request closed without merging**